pkgs/development/ocaml-modules/happy-eyeballs/mirage.nix

```diff
@@ -1,25 +1,39 @@
 { buildDunePackage
 , happy-eyeballs
+, duration
 , dns-client
+, domain-name
+, ipaddr
+, fmt
 , logs
 , lwt
+, mirage-clock
+, mirage-random
+, mirage-time
 , tcpip
 }:
 
 buildDunePackage {
   pname = "happy-eyeballs-mirage";
+
   inherit (happy-eyeballs) src version;
 
+  minimalOCamlVersion = "4.08";
+
+  strictDeps = true;
+
   propagatedBuildInputs = [
     dns-client
     happy-eyeballs
-    logs
     lwt
+    mirage-clock
+    mirage-random
+    mirage-time
     tcpip
   ];
+  doCheck = false;
```

Comment:
```suggestion
```
`doCheck = false;` is default.

Reply:
Unless someone plans on reverting that in the future.
Would make sense default to be true (in the future).
@vbgl Is there plans for that?

Reply:
Looks like we can run tests now so I enabled them